release_notes.md

```diff
@@ -0,0 +1,38 @@
+ # Release notes
```

Comment:
```suggestion
# Release notes
```

Reply:
We decided  with the team to to delete this information. In previous review you wrote that you also don't mind. Now the file release_notes.md keeps only the title of the file and the app name, because we haven't made any releases yet.